SPECS-EXTENDED/rdma-core/rdma-core.signatures.json

```
@@ -0,0 +1,5 @@
+{
+    "Signatures": {
+     "rdma-core-55.0.tar.gz": "6F8B97267807CDAE54845F542EE3D75DE80FDC24FE2632F5DB1573ECEF132D0F"
+    }
+   }
```

```suggestion
}
   
```
Nit - spacing & newline at EOF